app-modules/assist-data-model/database/migrations/sis/2023_07_17_200449_create_students_table.php

```diff
@@ -18,26 +18,27 @@ public function up(): void
                 $table->string('email')->nullable();
                 $table->string('email_2')->nullable();
                 $table->string('mobile')->nullable();
-                // TODO: Should sms_opt_out be a boolean?
-                $table->string('sms_opt_out')->nullable();
-                // TODO: Should email_bounce be a boolean?
-                $table->string('email_bounce')->nullable();
+                $table->boolean('sms_opt_out');
+                $table->boolean('email_bounce');
                 $table->string('phone')->nullable();
                 $table->string('address')->nullable();
-                $table->string('address_2')->nullable();
+                $table->string('address2')->nullable();
+                $table->string('address3')->nullable();
+                $table->string('city')->nullable();
+                $table->string('state')->nullable();
+                $table->string('postal')->nullable();
                 $table->date('birthdate')->nullable();
                 $table->integer('hsgrad')->nullable();
-                // TODO: Should dual be a boolean?
-                $table->string('dual')->nullable();
-                // TODO: Should ferpa be a boolean?
-                $table->string('ferpa')->nullable();
-                $table->float('gpa', 4, 3)->nullable();
-                $table->string('dfw')->nullable();
-                // TODO: Should firstgen be a boolean?
-                $table->string('firstgen')->nullable();
+                $table->boolean('dual')->nullable();
+                $table->boolean('ferpa')->nullable();
```

Comment:
Should the `nullable` `boolean` fields instead be defaulted to false?

Reply:
These actually don't make sense to be nullable, changed.